### DISCUSSION TITLE
MultiDexed: 8 Dexed instances in one plugin

### DISCUSSION THREAD

**Discussion by probonopd** [asb2m10/dexed#385] [Show and tell]
MultiDexed is a standalone application and a VST plugin for Windows, macOS, and Linux that runs multiple instances of [Dexed](https://github.com/asb2m10/dexed) to create the unison effect explained [here](https://www.youtube.com/watch?v=Hzwvd8aZUUU) and showcased [here](https://www.youtube.com/watch?v=BjxuzeVXCDU&t=1190s) by Anders Enger Jensen.

The instances can be detuned and stereo panned.

MultiDexed is especially useful in DAWs with a limited number of tracks, such as Ableton Live Lite.

It is __not a fork__ of Dexed. Rather, it hosts 8 instances of the Dexed VST3 (which must be installed).

https://github.com/probonopd/MultiDexed

**Comment by mtarenskeen** [comment]
Wow, cool. I was looking for something like this. I once did something like this by manually running 8 standalone versions of Dexed simultaneously (on Linux Fedora), because I wanted to try some SysEx patches that were created specifically for the Yamaha TX816. It sounded great, but the screen was a mess with all those open windows. I will try MultiDexed, it might be more convenient!

**Comment by probonopd** [comment]
Never got this to work reliably across all platforms and hosts. Probably I am missing something. If anyone wants to hack on it you are more than welcome to.
